Eli250/my-brand
# #jurc413q updating admin panel

#### What does this PR do?

Update admin panel

#### Description of Task to be completed?

This PR adds features to admin pane that were not present before.
#### How should this be manually tested?

This should be tested by checking if admin can view all created posts and see the total counts on dashboard.

#### What are the relevant pivotal tracker stories?

ZNbdHQuX

#### Screenshots:

![Screenshot from 2021-12-28 20-36-21](https://user-images.githubusercontent.com/60324751/147596645-7f7a78f3-234d-4d74-bc3f-dfa98af37d4f.png)
![Screenshot from 2021-12-28 20-36-26](https://user-images.githubusercontent.com/60324751/147596658-b0cd3b36-b4ef-41b3-bc10-84d8d0e57ea1.png)

#### Questions
N/A
:
